File: FancyRatProject_Part2/POO/class/Student.py
class Student:
    def __init__(self, name, surname, grade_level):
        self.name = name
        self.surname = surname
        self.grade_level = grade_level
        self.qualifications = {} # Dictionary to store the student's qualifications

    def add_qualifications(self, subject, qualifications):
        self.qualifications[subject] = qualifications

    def display_qualifications(self):
        for subject, qualifications in self.qualifications.items():
            print(f"{self.name}'s qualifications in {subject}:")
            for qual in qualifications.items():
                print(f"{qual[0]}: {qual[1]}")

File: FancyRatProject_Part2/web/java/Student.java
class Student {
    int id;
    String name;
    int age;
    String gender;

    public Student(int id, String name, int age, String gender) {
        this.id = id;
        this.name = name;
        this.age = age;
        this.gender = gender;
    }

    @Override
    public String toString() {
        return "ID: " + id + ", Name: " + name + ", Age: " + age + ", Gender: " + gender;
    }
}
